docs/tutorials/basics/50_control_flow.md

```diff
@@ -240,3 +240,136 @@ end
+Otherwise, it would continue forever or until you stop the program externally (for example <kbd>Ctrl+C</kbd>, `kill`, pull the plug or when armageddon arrives).
+
+In this example, not incrementing the counter it would be the same as writing:
+
+```
+while true
+  puts "Counter: #{counter}"
+end
+```
+Or if the condition was `counter > 0`, it would match for all values: they only increase from `1`.
+This would not technically be infinite, as it will fail with a math error when the counter reaches the maximum value of a 32-bit integer. But conceptually that's similar to an infinite loop.
+Such logic errors can be easy to miss and so it's very important to pay attention when writing the loop condition and also taking care of meeting said breaking case
+A good practice for index variables (such as `counter` in our example) is to increment them at the beginning of the loop.
+That makes it harder to forget that.
+
+!!! tip
+    Fortunately, there are many features in the language that relieve the burden of writing loops manually
+    and also take care of ensuring valid breaking conditions. A few of them will be introduced in following lessons.
+
+In some cases, the intention is to really have an endless loop.
+An example would be a server that always repeats waiting for a connection, or
+a command processor waiting for user input.
+Then it should be obvious of course, and not hidden in a complex, never-failing loop condition. The most plain way to express that is `while true`.
+The condition `true` is always truthy, so the loop repeats endlessly.
+
+```cr
+while true
+  puts "Hi, what's your name? (hit Enter when done)"
+
+  # `gets` returns input from the console
+  name = gets
+
+  puts "Nice to meet you, #{name}."
+  puts "Now, let's repeat."
+end
+```
+
+!!! note
+    This example is not an interactive playground by choice because the playground can't
+    handle non self-terminating programs, and processing user input.
+    It would just time out and print an error.
+    You can compile and run this code with a local compiler, though.
+
+    To stop the program, hit <kbd>Ctrl+C</kbd>. This sends a signal to the process asking it
+    to exit.
+
+### Skipping and Breaking
+
+It can be useful to skip some iterations in between, or stop the iteration entirely on some condition.
+
+The keyword `next` inside a loop body skips to the next iteration, ignoring any expressions left in the current iteration.
+If the loop condition isn't met, the loop finishes and the body won't execute another time.
+
+```{.crystal .crystal-play}
+counter = 0
+
+while counter < 10
+  counter += 1
+
+  if counter % 3 == 0
+    next
+  end
+
+  puts "Counter: #{counter}"
+end
+```
+
+This example could've easily be written without `next` by placing the `puts` expression in a conditional instead.
+The value of `next` becomes apparent when there are many more expressions in the method body to be skipped.
```

Comment:
"value" is used as the value of an expression, so it can get a bit confusing.
```suggestion
The worth of `next` becomes apparent when there are many more expressions in the method body to be skipped.
```